pkg/odo/cli/catalog/describe/operator_backend.go

```diff
@@ -108,30 +125,108 @@ func (ohb *operatorBackend) RunDescribeService(dso *DescribeServiceOptions) erro
 		return nil
 	}
 
+	svc := service.NewOperatorBackedService(ohb.Name, ohb.CR.Kind, ohb.CR.Version, ohb.CR.Description, ohb.CR.DisplayName, ohb.CRDSpec)
 	if log.IsJSON() {
-		machineoutput.OutputSuccess(service.ConvertCRDToJSONRepr(ohb.CR))
-		return nil
+		machineoutput.OutputSuccess(svc)
+	} else {
+		HumanReadableOutput(os.Stdout, svc)
 	}
-	repr := service.ConvertCRDToRepr(ohb.CR)
+	return nil
+}
 
-	fmt.Printf("Kind: %s\n", repr.Kind)
-	fmt.Printf("Version: %s\n", repr.Version)
-	fmt.Printf("Description: %s\n", repr.Description)
-	fmt.Println("Parameters:")
+func HumanReadableOutput(w io.Writer, service service.OperatorBackedService) {
+	fmt.Fprintf(w, "Kind: %s\n", service.Spec.Kind)
+	fmt.Fprintf(w, "Version: %s\n", service.Spec.Version)
+	fmt.Fprintf(w, "Description: %s\n", service.Spec.Description)
 
+	if service.Spec.Schema == nil {
+		log.Warning("Unable to get parameters from CRD or CSV")
```

Comment:
```suggestion
		log.Warningf("Unable to get parameters from CRD or CSV; Operator %q doesn't have the required information", service.Name)
```

Reply:
@feloy it's probably just me, but I feel like being explicit about pointing why odo couldn't get the parameters for the user.